src/includes/defines.h

```diff
@@ -72,12 +73,12 @@
 #define USERFLAGGED   "flagged"
 
 /* seemless reboot */
-#define REBOOTING_DIR           "reboot"
-#define USER_LIST_FILE          "reboot/_ulist"
-#define TALKER_SYSINFO_FILE     "reboot/_sysinfo"
-#define CHILDS_PID_FILE         "reboot/_child_pid"
-#define ROOM_LIST_FILE          "reboot/_rlist"
-#define LAST_USERS_FILE         "reboot/_last"
+#define REBOOTING_DIR           BASE_STORAGE_DIR "reboot"
```

Comment:
If BASE_STORAGE_DIR includes a final /, shy not the same on REBOOTING_DIR (or vice-versa, but making it consistent?)

Reply:
Nice spot!  I've made them more consistent now as of https://github.com/amnuts/Amnuts/pull/12/commits/bed438d8701a4eaa06f1cf1db2c83bf49c70c1fc